@uportal/content-carousel/src/components/ContentCarousel.html

```diff
@@ -2,11 +2,22 @@
   <template :v-if="computedItems.length">
     <slick ref="slick" :options="computedSlickOptions">
       <div v-for="item in computedItems" :key="item.id">
-        <a :href="item.destinationUrl">
+        <div v-if="computedSlickOptions.slidesToShow === 1" style="position: relative;">
+          <a :href="item.destinationUrl">
           <img v-if="item.imageUrl" :src="item.imageUrl" :alt="item.altText">
           <template v-if="!item.imageUrl">{{ item.title }}</template>
+          <div v-if="item.altText" style="width: 100%; position: absolute; bottom: 0; background: rgb(0,0,0,0.5); color: white;">
+            <p style="margin: 0; padding: 10px;" v-bind:style="alignText">{{ item.altText }}</p>
```

Comment:
I'm not sure `alignText` will do anything here.
it needs to know to bind to css `text-align`.
https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax-1

Reply:
This calls the alignText method that returns text-align property. When I change the property to "right" it justifies to the right.

Reply:
Hmm, Vue magic :tophat: :rabbit: :sparkles: :man_shrugging: